graf2d/x11ttf/CMakeLists.txt

```diff
@@ -28,4 +26,8 @@ ROOT_STANDARD_LIBRARY_PACKAGE(GX11TTF
   DEPENDENCIES
     GX11
     Graf
+  INCLUDES
+    ${FREETYPE_INCLUDE_DIRS} 
```

Comment:
These are not needed if we use `target_include_directories`: we inherit them from the library dependencies on GX11, Graf.

Reply:
Done